src/main/java/com/wcc/platform/domain/template/RenderedTemplate.java

```diff
@@ -0,0 +1,16 @@
+package com.wcc.platform.domain.template;
+
+import lombok.Data;
+
+@Data
```

Comment:
Probably here you could use:
```
@Getter
@NoArgsConstructor
@AllArgsConstructor
```

Also could be record because you are not using builder or anything else special.